backend/app/views/spree/admin/shared/_address_form.html.erb

```diff
@@ -1,80 +1,38 @@
-    <%= error_message_on f.object, :city %>
-  <% end %>
-
-  <%= field_container f.object, :zipcode, class: ["form-group", "#{type}-row"] do %>
-    <%= f.label :zipcode, Spree.t(:zip) %>
-    <%= f.text_field :zipcode, class: 'form-control' %>
-    <%= error_message_on f.object, :zipcode %>
-  <% end %>
-
-  <div class="form-group <%= "#{type}-row" %>">
-    <%= f.label :country_id, Spree.t(:country) %>
-    <span id="<%= s_or_b %>country">
-      <%= f.collection_select :country_id, available_countries, :id, :name, {}, { class: 'select2' } %>
-    </span>
-  </div>
-
-  <%= field_container f.object, :state, class: ["form-group", "#{type}-row"] do %>
-    <%= f.label :state_id, Spree.t(:state) %>
-    <span id="<%= s_or_b %>state">
-      <% if f.object.country.try(:states) %>
-
-      <%= f.text_field :state_name,
-            style: "display: #{f.object.country.try(:states) && f.object.country.states.empty? ? 'block' : 'none' };",
-            disabled: !(f.object.country.try(:states) && f.object.country.states.empty?), class: 'form-control state_name' %>
-        <%= f.collection_select :state_id, f.object.country.states.sort, :id, :name, { include_blank: true }, { class: 'select2', style: "display: #{ f.object.country.states.empty? ? 'none' : 'block' };", disabled: f.object.country.states.empty? } %>
-      <% else %>
-        <div>
-          <em>No country set.</em>
-        </div>
-      <% end %>
+<% Spree::Address::ADDRESS_FIELDS.each do |field| %>
+  <% if field == "country" %>
+    <div class="form-group <%= "#{type}-row" %>">
+      <%= f.label :country_id, Spree.t(:country) %>
+      <span id="<%= s_or_b %>country">
+        <%= f.collection_select :country_id, available_countries, :id, :name, {}, { class: 'select2' } %>
       </span>
-    <%= error_message_on f.object, :state_id %>
-  <% end %>
-
-  <%= field_container f.object, :phone, class: ["form-group", "#{type}-row"] do %>
-    <%= f.label :phone, Spree.t(:phone) %>
-    <%= f.phone_field :phone, class: 'form-control' %>
-    <%= error_message_on f.object, :phone %>
+    </div>
+  <% elsif field == "state" %>
+    <%= field_container f.object, :state, class: ["form-group", "#{type}-row"] do %>
+      <%= f.label :state_id, Spree.t(:state) %>
+      <span id="<%= s_or_b %>state">
+        <% if f.object.country.try(:states) %>
+          <%= f.text_field :state_name,
+                           style: "display: #{f.object.country.try(:states) && f.object.country.states.empty? ? 'block' : 'none' };",
+                           disabled: !(f.object.country.try(:states) && f.object.country.states.empty?), class: 'form-control state_name' %>
+          <%= f.collection_select :state_id, f.object.country.states.sort, :id, :name, { include_blank: true }, { class: 'select2', style: "display: #{ f.object.country.states.empty? ? 'none' : 'block' };", disabled: f.object.country.states.empty? } %>
+        <% else %>
+          <div>
+            <em>No country set.</em>
```

Comment:
@imgarylai can we use translation here? Thank you!

Reply:
Sure. Thanks for reminding me.